java/server/test/org/openqa/selenium/netty/server/NettyServerTest.java

```diff
@@ -62,7 +61,7 @@ public void ensureMultipleCallsWorkAsExpected() {
       ).start();
 
       // Very deliberately uses OkHttp.
-      HttpClient client = new OkHttpClient.Factory().createClient(server.getUrl());
+      HttpClient client = (HttpClient) HttpClient.Factory.create(server.getUrl().toString());
```

Comment:
This cast is wrong. `create` returns a new http client factory (not a client), and we use the string to figure out the name of the factory to use ("okhttp", "netty", "reactor"). The server url gets passed to _that_ in order to create the httpclient itself.